{
    "object_alias": "exface.Core.MESSAGE",
    "columns": [
        {
            "name": "_EXPORT_SUMMARY",
            "hidden": true,
            "attribute_alias": "LABEL"
        },
        {
            "name": "CREATED_ON",
            "attribute_alias": "CREATED_ON"
        },
        {
            "name": "MODIFIED_ON",
            "attribute_alias": "MODIFIED_ON"
        },
        {
            "name": "CREATED_BY_USER",
            "attribute_alias": "CREATED_BY_USER"
        },
        {
            "name": "MODIFIED_BY_USER",
            "attribute_alias": "MODIFIED_BY_USER"
        },
        {
            "name": "UID",
            "attribute_alias": "UID"
        },
        {
            "name": "APP",
            "attribute_alias": "APP"
        },
        {
            "name": "CODE",
            "attribute_alias": "CODE"
        },
        {
            "name": "TITLE",
            "attribute_alias": "TITLE"
        },
        {
            "name": "DESCRIPTION",
            "attribute_alias": "DESCRIPTION"
        },
        {
            "name": "HINT",
            "attribute_alias": "HINT"
        },
        {
            "name": "TYPE",
            "attribute_alias": "TYPE"
        },
        {
            "name": "DOCS",
            "attribute_alias": "DOCS"
        }
    ],
    "rows": [
        {
            "_EXPORT_SUMMARY": "6T4RAVX Failed to create HTTP client",
            "CREATED_ON": "2016-12-20 07:55:40",
            "MODIFIED_ON": "2016-12-20 07:55:40",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11e6c689a72e43e9a3e40205857feb80",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6T4RAVX",
            "TITLE": "Failed to create HTTP client",
            "DESCRIPTION": "",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "_EXPORT_SUMMARY": "6UY4L3E Incorrect request to a webservice",
            "CREATED_ON": "2017-04-26 09:47:00",
            "MODIFIED_ON": "2017-08-22 15:22:58",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11e72a654b1c523f8f0674e5434dc47d",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6UY4L3E",
            "TITLE": "Incorrect request to a webservice",
            "DESCRIPTION": "This error results from an incorrect request to a webservice. Please contact the technical support. Don't forget to communicate the error code!",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "_EXPORT_SUMMARY": "6UY5F5D Authentication to a webservice failed",
            "CREATED_ON": "2017-04-26 10:51:55",
            "MODIFIED_ON": "2017-08-22 15:31:36",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11e72a6e5cb4394d8f0674e5434dc47d",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6UY5F5D",
            "TITLE": "Authentication to a webservice failed",
            "DESCRIPTION": "The authentication to a webservice failed. You're not allowed to access the requested resource. Please contact the technical support. Don't forget to communicate the error code!",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "_EXPORT_SUMMARY": "6UY86E7 Internal error in webservice",
            "CREATED_ON": "2017-04-26 14:26:17",
            "MODIFIED_ON": "2017-08-22 15:21:49",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11e72a8c4ee7c3178f0674e5434dc47d",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6UY86E7",
            "TITLE": "Internal error in webservice",
            "DESCRIPTION": "This error results from an internal error in a webservice. Please contact the technical support. Don't forget to communicate the error code!",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "_EXPORT_SUMMARY": "6UY88MN Unknown error during request to a webservice",
            "CREATED_ON": "2017-04-26 14:31:07",
            "MODIFIED_ON": "2017-08-22 15:19:47",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11e72a8cfb8b80de8f0674e5434dc47d",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6UY88MN",
            "TITLE": "Unknown error during request to a webservice",
            "DESCRIPTION": "An unknown error has occured during a request to a webservice. Please contact the technical support. Don't forget to communicate the error code!",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "_EXPORT_SUMMARY": "6UY95WR No response from webserver",
            "CREATED_ON": "2017-04-26 15:43:00",
            "MODIFIED_ON": "2017-08-22 15:20:43",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11e72a9706685b938f0674e5434dc47d",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6UY95WR",
            "TITLE": "No response from webserver",
            "DESCRIPTION": "This error occurs if a webserver can't be contacted for some reason. Please contact the technical support. Don't forget to communicate the error code!",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "_EXPORT_SUMMARY": "6V27IZ3 The webservice could not be contacted",
            "CREATED_ON": "2017-05-04 08:13:08",
            "MODIFIED_ON": "2017-08-22 15:21:23",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11e730a182a8a06fb40774e5434dc47d",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6V27IZ3",
            "TITLE": "The webservice could not be contacted",
            "DESCRIPTION": "This error occurs when a webservice could not be contacted at the URL specified in the current connection. Please contact the technical support. Don't forget to communicate the error code!",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "_EXPORT_SUMMARY": "73G87II Cannot convert oData 2.0 association to a relation in the metamodel",
            "CREATED_ON": "2018-12-12 11:12:52",
            "MODIFIED_ON": "2018-12-12 11:23:56",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11e8fdfed8ded899a23ee4b318306b9a",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "73G87II",
            "TITLE": "Cannot convert oData 2.0 association to a relation in the metamodel",
            "DESCRIPTION": "",
            "HINT": "Try to re-run the model builder for the objects at both ends of the association node",
            "TYPE": "ERROR",
            "DOCS": "exface/UrlDataConnector/Docs/OData/The_metamodel_for_OData_2.0.md"
        },
        {
            "_EXPORT_SUMMARY": "75C7YOQ Missing parameter for remote function call",
            "CREATED_ON": "2019-04-23 15:30:16",
            "MODIFIED_ON": "2019-04-23 15:42:34",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11e965dc7ff50a18812a0205857feb80",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "75C7YOQ",
            "TITLE": "Missing parameter for remote function call",
            "DESCRIPTION": "",
            "HINT": "Please include a column named exactly as the parameter in the input data (e.g. by using an input_mapper)!",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "_EXPORT_SUMMARY": "76G66CL Invalid GraphQL mutation configuration",
            "CREATED_ON": "2019-07-10 07:25:42",
            "MODIFIED_ON": "2019-07-10 07:25:42",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11e9a2e3eb77995fac8e0205857feb80",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "76G66CL",
            "TITLE": "Invalid GraphQL mutation configuration",
            "DESCRIPTION": [
                "// Multiline text delimited by `\n`",
                "This error indicates, that a generic modifying action (create, update or delete) is performed upon an object from a GraphQL data source, but the configuration of that object does not include the mutation name for that action. ",
                "",
                "The GraphQL query builder needs to know, which mutations to perform for these generic actions. The corresponding mutation names must be defined in the meta object's data address properties to make these actions work:",
                "",
                "- graphql_create_mutation",
                "- graphql_update_mutation",
                "- graphql_delete_mutation"
            ],
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "_EXPORT_SUMMARY": "7EO5HDU Stored login data not valid anymore",
            "CREATED_ON": "2021-02-04 17:42:20",
            "MODIFIED_ON": "2021-02-04 17:42:20",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11ebb579f36c8c76b579025041000001",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "7EO5HDU",
            "TITLE": "Stored login data not valid anymore",
            "DESCRIPTION": [
                "// Multiline text delimited by `\n`",
                "Some web services will block users if they fail to login multiple times in a row. To prevent this, HTTP data connectors remember the credentials that failed and ask the user to re-login. ",
                "",
                "This behavior can be explicitly turned on or off via `authentication_retry_after_fail` in the configuration of the data connection."
            ],
            "HINT": "Please log in again manually!",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "_EXPORT_SUMMARY": "7STUR1D Invalid metamodel URI",
            "CREATED_ON": "2023-10-23 15:27:05",
            "MODIFIED_ON": "2023-10-23 15:27:05",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11ee8697dc520cfc8697025041000001",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "7STUR1D",
            "TITLE": "Invalid metamodel URI",
            "DESCRIPTION": [
                "// Multiline text delimited by `\n`",
                "This error occurs when URL query builders are used on data stored in other data sources, but the URL is not correctly formed. The expected URL format is `metamodel://my.APP.OBJECT/<attribute_alias>/<uid_value>`, where",
                "",
                "- `my.APP.OBJECT` is the fully qualified alias of the meta object to read or write",
                "- `<attribute_alias>` is the alias of the attribute, that is expected to contain the request body",
                "- `<uid_value>` is the UID of the target data row",
                "",
                "GET-requests will read data, while POST, PUT and PATCH will write data. DELETE will delete the entire row."
            ],
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        }
    ],
    "totals_rows": [],
    "filters": {
        "operator": "AND",
        "base_object_alias": "exface.Core.MESSAGE",
        "conditions": [
            {
                "expression": "APP",
                "comparator": "=",
                "value": "0x11e6951dda3ba520b48b847beb4a5184",
                "object_alias": "exface.Core.MESSAGE"
            }
        ]
    },
    "rows_limit": null,
    "rows_offset": 0,
    "sorters": [
        {
            "attribute_alias": "CREATED_ON",
            "direction": "ASC"
        },
        {
            "attribute_alias": "UID",
            "direction": "ASC"
        }
    ]
}